models/posts.model.js

```diff
@@ -0,0 +1,51 @@
+import Promise from 'bluebird';
+import mongoose, { Schema } from 'mongoose';
+import httpStatus from 'http-status';
+
+/**
+ * Post Schema
+ */
+const PostSchema = new Schema({
+  title: { type: String, required: true },
+  content: { type: String, required: true },
+  user: { type: Schema.Types.ObjectId, ref: 'User', required: true }
+}, {
+  timestamps: true
+});
+
+/**
+ * Add your
+ * - pre-save hooks
+ * - validations
+ * - virtuals
+ */
+
+/**
+ * Methods
+ */
+PostSchema.method({
+});
+
+/**
+ * Statics
+ */
+PostSchema.statics = {
+  /**
+   * List posts in descending order of 'createdAt' timestamp.
+   * @param {number} skip - Number of posts to be skipped.
+   * @param {number} limit - Limit number of posts to be returned.
+   * @returns {Promise<post[]>}
+   */
+  list({ skip = 0, limit = 50 } = {}) {
```

Comment:
`list({ skip = 0, limit = 50 } = {}) {
    return this.find()
      .sort({ createdAt: -1 })
      .skip(+skip)
      .limit(+limit)
      .exec()
      .then(users => ({
        limit: parseInt(limit),
        skip: parseInt(skip),
        data: users
      }));
  }`

Nên trả về dạng thế này, a đang sửa ở bên phần users model của anh, em sửa lại luôn theo thế này đi, trả về metadata của pagination để client dễ xử lý.

Reply:
updated